errors/errors.go

```diff
@@ -20,13 +20,13 @@
 
 package errors
 
-// RuleConflictError is returned when a rule modification is made that would
+// InvalidInputError is returned when a rule modification is made that would
 // result in a conflict with the existing set of rules in the ruleset.
-type RuleConflictError string
+type InvalidInputError string
```

Comment:
FYI we'll need to update the switch statement in `m3ctl` to reflect this change.

Reply:
Yep. Ill update the existing m3ctl diff when this has been landed and imported.